Library/Homebrew/cask/lib/hbc/cli.rb

```diff
@@ -230,7 +230,7 @@ def run(*_args)
 
         return if @command == "help" && @args.empty?
 
-        unknown_command = @args.empty? ? @command : @args.first
+        unknown_command = @command == "help" ? @args.first : @command
```

Comment:
CC @reitermarkus for a 👍 on this.

Reply:
This error is a bit useless actually, because `brew cask help install` will result in `Error: Unknown command: install`, because there is no per-command help implemented.

I think the error should be `"help does not take arguments."`.